test/e2e/operator_test.go

```diff
@@ -88,9 +90,11 @@ func TestDefaultDNSExists(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	name := types.NamespacedName{Name: "default"}
 	err = wait.PollImmediate(1*time.Second, 5*time.Minute, func() (bool, error) {
 		dns := &operatorv1.DNS{}
-		if err := cl.Get(context.TODO(), types.NamespacedName{Name: "default"}, dns); err != nil {
+		if err := cl.Get(context.TODO(), name, dns); err != nil {
+			t.Logf("failed to get dns operator %s: %v", name, err)
```

Comment:
The log msg will look better with:
`t.Logf("failed to get deployment %s: %v", namespacedName.Name, err)`